scripts/configure-ironic.sh

```diff
@@ -87,7 +88,7 @@ if [ ! -z "${IRONIC_EXTERNAL_IP}" ]; then
 	else
 		export IRONIC_EXTERNAL_CALLBACK_URL="http://${IRONIC_EXTERNAL_IP}:6385"
 	fi
-	export IRONIC_EXTERNAL_HTTP_URL="http://${IRONIC_EXTERNAL_IP}:6180"
+	export IRONIC_EXTERNAL_HTTP_URL="http://${IRONIC_EXTERNAL_IP}:${HTTP_PORT}"
```

Comment:
Comment for other reviewers: HTTP_PORT=6180 is what is hardcoded in cluster-baremetal-operator now, but it can be changed. Hence the need to use HTTP_PORT.